packages/customize-widgets/src/components/sidebar-block-editor/sidebar-adapter.js

```diff
@@ -59,32 +43,21 @@ export default class SidebarAdapter {
 		];
 		this.historyIndex = 0;
 
-		this._handleSettingChange = this._handleSettingChange.bind( this );
-		this._handleAllSettingsChange = this._handleAllSettingsChange.bind(
-			this
-		);
+		this.setting.bind( this._handleSettingChange.bind( this ) );
+		this.api.bind( 'change', this._handleAllSettingsChange.bind( this ) );
```

Comment:
Move to constructor so that it can still listen to the changes even when it's not active yet.